docs/about/whats_new.rst

```diff
@@ -8,6 +8,11 @@ What's New
 Next Release
 ============
 
+What's Changed
+--------------
+* Extend and integrate `native_load` :pull:`2053`
```

Comment:
We no longer ask for core PRs to edit whats_new.rst

Github can auto-generated release notes based on PR titles - so just make sure the PR title is suitable for use in release notes  (this one is fine).